stylesheets/_component.tooltips.scss

```diff
@@ -32,7 +32,8 @@
   color: $tooltip-color;
   text-align: center;
   text-decoration: none;
-  background-color: $tooltip-bg;
+  background-color: rgba(0, 0, 0, .9);
```

Comment:
Could we use `rgba(color(black), .9)` here instead, which would make it consistent with the rest of the framework that uses an off-black shade for wherever black is needed?